modules/sagemaker/sagemaker-model-monitoring/README.md

```diff
@@ -24,19 +25,57 @@ for each monitoring job:
 - `kms-key-id`: The KMS key used to encrypted storage and output.
-- `data-quality-checkstep-output-prefix`: The S3 prefix in `model-artifacts-bucket-arn` which contains the output from the corresponding [Check step in the SageMaker Pipeline](https://docs.aws.amazon.com/sagemaker/latest/dg/build-and-manage-steps.html#build-and-manage-steps-types).
-- `data-quality-output-prefix`: The S3 prefix in `model-artifacts-bucket-arn` to contain the output of the monitoring job.
 
+One or more of:
+
+- `enable-data-quality-monitor`: True to enable the data quality monitoring job.
+- `enable-model-quality-monitor`: True to enable the model quality monitoring job.
 
 #### Optional
 
 - `sagemaker-project-id`: SageMaker project id
 - `sagemaker-project-name`: SageMaker project name
-- `data-quality-instance-count`: The number of ML compute instances to use in the model monitoring job.
-- `data-quality-instance-type`: The ML compute instance type for the processing job.
-- `data-quality-instance-volume-size-in-gb`: The size of the ML storage volume, in gigabytes, that you want to provision.
-- `data-quality-max-runtime-in-seconds`: The maximum length of time, in seconds, the monitoring job can run before it is stopped.
-- `data-quality-schedule-expression`: A cron expression that describes details about the monitoring schedule.
+
+### Per-job Parameters
+
+Each job has prefixed parameters, some of which are required. E.g. if
+`enable-data-quality-monitor` is set to `true`, then the
+`data-quality-checkstep-output-prefix` parameter is required.
+
+#### Required
+
+- `*-checkstep-output-prefix`: The S3 prefix in `model-artifacts-bucket-arn` which contains the output from the corresponding [Check step in the SageMaker Pipeline](https://docs.aws.amazon.com/sagemaker/latest/dg/build-and-manage-steps.html#build-and-manage-steps-types).
+- `*-output-prefix`: The S3 prefix in `model-artifacts-bucket-arn` to contain the output of the monitoring job.
+
+#### Optional
+
+- `*-instance-count`: The number of ML compute instances to use in the model monitoring job.
+- `*-instance-type`: The ML compute instance type for the processing job.
+- `*-instance-volume-size-in-gb`: The size of the ML storage volume, in gigabytes, that you want to provision.
+- `*-max-runtime-in-seconds`: The maximum length of time, in seconds, the monitoring job can run before it is stopped.
+- `*-schedule-expression`: A cron expression that describes details about the monitoring schedule.
+
+#### Job-specific Parameters
+
+See below for the specific input parameters for each model monitoring
+job. These parameters are all required.
+
+##### Data Quality Monitoring Job Parameters
+
+N/A
+
+##### Model Quality Monitoring Job Parameters
+
+###### Required
+
+- `ground-truth-prefix`: The S3 prefix in `model-artifacts-bucket-arn` which contains the [ground truth](https://docs.aws.amazon.com/sagemaker/latest/dg/model-monitor-model-quality-merge.html) for captured data.
```

Comment:
This is not prefixed since I plan to re-use it for the model explainability monitor as well.